tests/test_scripts.py

```diff
@@ -1,22 +1,27 @@
 import os
+import json
 from entity_resolver.scripts import run
 
 
-class TestArxiv:
+class TestScripts:
 
-    def test_dummy(self):
-        file_path = 'file.txt'
-        run(['norm-arxiv', '', file_path])
-        with open(file_path, 'r') as f:
-            assert f.read() == 'arxiv'
-        os.remove(file_path)
+    def test_citeseer(self):
+        file_path = 'data/citeseer/citeseer-mrdm05.dat'
+        output_path = 'testdata.json'
+        run(['norm-citeseer', file_path, output_path])
+        with open(output_path, 'r') as f:
+            test_dict = json.load(f)
 
+        assert test_dict[0]['attr_dict']['name'] == '  A. Aamodt '
+        assert test_dict[0]['node_id'] == 0
+        assert test_dict[0]['edge_id'] == 1019
```

Comment:
Use `os.remove(output_path)` after the assert statements to remove the test json file after the test is done. Same applies to the `test_arxiv` function.